src/locales/en/translation.json

```
@@ -621,26 +627,51 @@
   "management_regimes": {
     "management_regime": "Management regime",
     "management_regimes": "Management Regimes",
+    "management_regime_low": "management regime",
     "management_regime_name": "Management regime name",
     "management": "Management",
     "secondary_name": "Secondary name",
     "year_est": "Year est.",
+    "area": "Area",
+    "parties": "Parties",
     "compliance": "Compliance",
+    "not_reported": "Not reported",
     "open_access": "Open access",
     "access_restriction": "Access restrictions",
+    "partial_restrictions": "Partial restrictions: {{restrictions}}",
+    "partial_restrictions_label": "Partial restrictions",
```

Suggestion: 
```suggestion
    "partial_restrictions": "Partial restrictions: {{restrictions}}",
    "partial_restrictions_label": "Partial restrictions",
```
```suggestion
    "partial_restrictions": "Partial restrictions",
```
Remove `partial_restrictions_label` and replace `partial_restrictions` with that text. It looks like any restrictions passed through should have their own tokenization